modules/GephiSimilarityComputer/src/main/java/Controller/MyImporterBuilder.java

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Also please remove the empty template header from all files and make sure authorship is correct in each file.